ci-operator/step-registry/ofcir/acquire/ofcir-acquire-commands.sh

```diff
@@ -120,8 +120,12 @@ function getCIR(){
     fi
 }
 
-#CLUSTERTYPE can be one of "virt", "virt-arm64" or "baremetal"
-CIRTYPE=host
+# Most virt based jobs run on all of the CI hosts, but the diask space available
+# in ESI isn't enough for upgrade jobs
+CIRTYPE=host,host_esi
+[[ "$JOB_NAME" =~ -upgrade- ]] && CIRTYPE=host
```

Comment:
CIRTYPE looks like it’s getting set below, does this get overridden?

Reply:
Yes, if the job is baremetal then it will be overridden, this is expected and also avoids the ESI nodes (they are in our lab)

For "baremetal-moc" CLUSTERTYPE, this only effects one okd job, which we will probably remove soon as its not being maintained.